server/requirements.txt

```diff
@@ -1,15 +1,21 @@
+black==20.8b1
 flask==1.1.2
 flask-cors==3.0.8
 flask_restful==0.3.8
 gensim==3.8.2
```

Comment:
can you also upgrade `gensim` to `3.8.3` (which is a bugfix release)? See https://github.com/RaRe-Technologies/gensim/releases/tag/3.8.3